examples/c/task_iter.h

```diff
@@ -0,0 +1,16 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+/* Copyright (c) 2020 Facebook */
+
+#define TASK_COMM_LEN 16
+#define MAX_STACK_LEN 127
+
+struct task_iter {
```

Comment:
nit: maybe name it something more meaningful, like "task_info" or something? iter doesn't make much sense here

Reply:
task_info works for me